forge-gui/res/cardsfolder/a/archangel_of_strife.txt

```diff
@@ -4,10 +4,9 @@ Types:Creature Angel
 PT:6/6
 K:Flying
 K:ETBReplacement:Other:ChooseEach
-SVar:ChooseEach:DB$ RepeatEach | RepeatPlayers$ Player | RepeatSubAbility$ DBChoice | SubAbility$ DBRemember | ClearRememberedBeforeLoop$ True | SpellDescription$ As CARDNAME enters the battlefield, each player chooses war or peace. Creatures controlled by players who chose war get +3/+0. Creatures controlled by players who chose peace get +0/+3.
-SVar:DBChoice:DB$ GenericChoice | Defined$ Player.IsRemembered | Choices$ WarChoice,PeaceChoice | ShowChoice$ ExceptSelf
-SVar:WarChoice:DB$ Pump | Defined$ Player.IsRemembered | NoteCards$ Self | NoteCardsFor$ Strife | SpellDescription$ War
-SVar:PeaceChoice:DB$ Pump | Defined$ Player.IsRemembered | ClearNotedCardsFor$ Strife | SpellDescription$ Peace
+SVar:ChooseEach:DB$ GenericChoice | Defined$ Player | TempRemember$ Chooser | Choices$ WarChoice,PeaceChoice | ShowChoice$ ExceptSelf | SpellDescription$ As CARDNAME enters the battlefield, each player chooses war or peace.
+SVar:WarChoice:DB$ Pump | Defined$ Remembered | NoteCards$ Self | NoteCardsFor$ Strife | SpellDescription$ War
+SVar:PeaceChoice:DB$ Pump | Defined$ Remembered | ClearNotedCardsFor$ Strife | SpellDescription$ Peace
 SVar:DBRemember:DB$ Pump | RememberObjects$ Player.NotedForStrife | ImprintCards$ Self
 S:Mode$ Continuous | AffectedZone$ Battlefield | Affected$ Creature.ControlledBy Remembered | AddPower$ 3 | Description$ Creatures controlled by players who chose war get +3/+0.
 S:Mode$ Continuous | AffectedZone$ Battlefield | Affected$ Creature.!ControlledBy Remembered | AddToughness$ 3 | IsPresent$ Card.IsImprinted | Description$ Creatures controlled by players who chose peace get +0/+3.
```

Comment:
Can the static below work without Remember/Imprint?

`Creature.ControlledBy Player.NotedForStrive` ?